packages/dx-react-grid/src/plugins/table-group-row.jsx

```diff
@@ -119,6 +222,60 @@ export class TableGroupRow extends React.PureComponent {
             </TemplateConnector>
           )}
         </Template>
+        <Template
+          name="tableCell"
+          predicate={({
+            tableRow, tableColumn,
+          }) => (isGroupRowOrdinaryCell(tableRow, tableColumn))}
+        >
+          {params => (
+            <TemplateConnector>
+              {(
+                {
+                  groupSummaryItems, groupSummaryValues, grouping, tableColumns,
+                },
+                { toggleGroupExpanded },
+              ) => {
+                const { tableColumn, tableRow } = params;
+                const onToggle = () => toggleGroupExpanded({ groupKey: tableRow.row.compoundKey });
+
+                if (isRowSummaryCell(tableRow, tableColumn, grouping, groupSummaryItems)) {
+                  const columnSummaries = getColumnSummaries(
+                    groupSummaryItems,
+                    tableColumn.column.name,
+                    groupSummaryValues[tableRow.row.compoundKey],
+                    summaryItem => summaryItem.showInGroupRow,
+                  );
+
+                  return (
+                    <RowSummaryCell
+                      {...params}
+                      row={params.tableRow.row}
+                      column={params.tableColumn.column}
+                      onToggle={onToggle}
+                    >
+                      <TableSummaryContent
```

Comment:
I think we should avoid dependency across plugins. The TableGroupRow plugin should not has summary templates.

If we want to pass summary's values to the grouping feature, we should override the grouping table cell template inside the TableSummaryRow plugin

Reply:
This will in fact increase coupling: for example if we want to show a summary in a group row we will have to add a SummaryRow plugin, now we need just SummaryState and GroupRow plugins. 
The main idea is that plugin that renders some part of UI, should not affect other UI parts

Reply:
Templates (dx-react-core) have been implemented for override UI across the plugins.
Your way complex for customization because a user should know what part of ui will be changed after change plugin's templates.

I think we should combine all templates for one feature in the one UI plugin.